{
  "type": "relationship",
  "id": "relationship--0d6df769-7a34-4642-85f5-bf577973456e",
  "created": "2021-10-14T20:28:09.822Z",
  "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
  "description": "(Citation: Unit42 Clop April 2021)(Citation: Cybereason Clop Dec 2020)",
  "external_references": [
    {
      "description": "Santos, D. (2021, April 13). Threat Assessment: Clop Ransomware. Retrieved July 30, 2021.",
      "source_name": "Unit42 Clop April 2021",
      "url": "https://unit42.paloaltonetworks.com/clop-ransomware/"
    },
    {
      "description": "Cybereason Nocturnus. (2020, December 23). Cybereason vs. Clop Ransomware. Retrieved May 11, 2021.",
      "source_name": "Cybereason Clop Dec 2020",
      "url": "https://www.cybereason.com/blog/cybereason-vs.-clop-ransomware"
    }
  ],
  "modified": "2021-10-14T20:28:09.822Z",
  "object_marking_refs": [
    "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
  ],
  "relationship_type": "uses",
  "source_ref": "intrusion-set--7eda3dd8-b09b-4705-8090-c2ad9fb8c14d",
  "spec_version": "",
  "target_ref": "malware--cad3ba95-8c89-4146-ab10-08daa813f9de",
  "x_mitre_attack_spec_version": "",
  "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
  "x_mitre_version": "1.0"
}
